support/dynamics/gp/guidelines-generate-eft-files.md

```diff
@@ -194,93 +194,85 @@ Q1: When does a support case for EFT assistance become a consulting expense? How
 
-Q9: What if my bank uses a file format that varies slightly from one of the standard formats offered in GP?
+To activate EFT, select **Tools** under the Microsoft Dynamics GP menu, point to **Setup**, point to **System** and select **Registration**. Make sure the appropriate module of EFT Payables or EFT Receivables is marked as appropriate.
 
-A9: The EFT file formats available in Microsoft Dynamics GP are set up to follow the US Government and NACHA (National Automated Clearing House Association) published formats. They're published standard requirements. The file formats currently offered in Microsoft Dynamics GP are coded to achieve the required **** fields in these standard formats. So if your bank requires a format that varies slightly from one of the standard formats offered in GP, or a more updated version of a format, you may need to pursue a consulting service for the customization. In the meantime, you could check with the bank to see if they would make an exception, or accept any other formats that Dynamics GP can do (such as one of the standard NACHA formats), or edit the EFT file in Notepad before sending it to the bank.
+To verify it's working for the respective module, select **Cards**, point to **Purchasing** (or **Sales**), and select **Vendor** (or **Customer**). Enter a Vendor ID (or Customer ID) and select the **Address** button. If you see a button named **EFT BANK** in the lower right corner of the Vendor (or Customer) Address Maintenance window, then EFT is working.
 
-You can search the [Microsoft ISV Central](https://www.microsoft.com/licensing/licensing-programs/isv-program) to find an ISV that may offer an EFT product compatible with Dynamics GP or could help you achieve a customization to meet your needs:
+Q8: What if my bank uses a file format that varies slightly from one of the standard formats offered in GP?
 
-Also enter a Product Suggestion in the [Ideas](https://experience.dynamics.com/ideas/) link so Microsoft can track it and consider this new functionality for a future enhancement.
+A8: The EFT file formats available in Microsoft Dynamics GP are set up to follow the US Government and NACHA (National Automated Clearing House Association) published formats. They're published standard requirements. The file formats currently offered in Microsoft Dynamics GP are coded to achieve the required **** fields in these standard formats. So if your bank requires a format that varies slightly from one of the standard formats offered in GP, or a more updated version of a format, you may need to pursue a consulting service for the customization. In the meantime, you could check with the bank to see if they would make an exception, or accept any other formats that Dynamics GP can do (such as one of the standard NACHA formats), or edit the EFT file in Notepad before sending it to the bank.
```

Comment:
Hi @theley502, can you help double check the "****" in line 229? Do we need to remove it? Thanks!

Reply:
looks good